### ISSUE TITLE
Cannot use conda's emscripten

### ISSUE THREAD

**Issue by gabrielfougeron** [pyodide/pyodide-build#50]
Hi,

Installing emscripten is a bit of a hassle. I've done it, it works with pyodide-build, but I'm trying to simplify the build process of my app and switch to conda-forge's distribution of emscripten.
I'm running pyodide 0.26.3 and pyodide-build 0.29.0 which require emscipten 3.1.58. Fortunately, it's available on conda-forge (pheww!).

I install everything, try to pyodide-build and get hit with the following message:

```bash
$ pyodide build
Incorrect Emscripten version 3.1.58-git. Need Emscripten version 3.1.58
```

I can't tell whether this is by design or a bug.

**Comment by agriyakhetarpal** [comment]
I use the one from Homebrew, and it lists the version with the "-git" suffix, too, so we should allow it in `installed_version`/`needed_version`.

**Comment by hoodmane** [comment]
Though installing with `emsdk` is really easy, I exclusively use that version:
```sh
git clone https://github.com/emscripten-core/emsdk.git --depth 1
./emsdk/emsdk install 3.1.58
./emsdk/emsdk activate 3.1.58
source ./emsdk/emsdk_env.sh
```

**Comment by agriyakhetarpal** [comment]
This should have been fixed with [`249f365`](https://github.com/pyodide/pyodide-build/pull/53/commits/249f365a277be3562c39d24472cde7abff8d35e4) via #53. Thanks for reporting!
